GithubJobApp/UINetImageView.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  UINetImageView.swift
+//  GithubJobApp
+//
+//  Created by Daniel Harrison on 12/1/16.
+//  Copyright © 2016 Taylor Harrison. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIImageView {
+    
+    func setImgFromUrl(url: NSURL) {
+        if let image = NetImageCache.cache.objectForKey(url) as? UIImage {
+            print("using cached image")
+            self.image = image
+        } else {
+            dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0)) {
+                dispatch_async(dispatch_get_main_queue()) { () -> Void in
```

Comment:
whoops, meant to create the placeholder UIImage object in the background thread, then put it into the UIImageView on the main thread